### PR TITLE
feat: link styling  `/about` team members

### PR DESCRIPTION
**Pull request by CTAPCKPIM**

### CODE REVIEW

**Comment by CTAPCKPIM** [comment]
@georgeciubotaru 

Tell me how I can achieve the goal, I tried different things and so far I have not been able to complete the task

image

When I remove the second check mark, the style becomes as it should, but I have not yet found where these lines are, I will be grateful for the hint

**Comment by georgeciubotaru** [comment]
@CTAPCKPIM 

You are on the good path

1. If you will check the `BodyRenderer/item.svelte` file you will find global styles applied to `b|em|strong` that also override the `a` elements text.

I think you get what you need to do next.

**Comment by CTAPCKPIM** [comment]
@georgeciubotaru 
- Done, check pls

**Comment by georgeciubotaru** [comment]
@CTAPCKPIM Not the correct version.

Try to identify what you need to do:

- We have `b` elements that have white text color, there can be around the project (different elements)
- You need to change that text-color for inline elements inside a link.

File: src/lib/components/BodyRenderer/inline.svelte [hunk 1]
```diff
@@ -27,8 +27,12 @@
...
-	:global(b),
-	:global(em),
-	:global(strong)
-		@apply text-t1
+  :global(b),
+  :global(em),
+  :global(strong)
+    @apply text-t1
+
+  :global(a),
```
Comment (georgeciubotaru) [inline src/lib/components/BodyRenderer/inline.svelte:35]: you have `Link` component where you can please this code.

remove comments above

File: src/lib/components/BodyRenderer/Blocks/link.svelte [hunk 2]
```diff
@@ -55,3 +55,9 @@
     <Icon icon={ArrowTopRightOnSquare} width={16} height={16} colorInherit />
   {/if}
 </a>
+
+<style lang="sass">
+  :global(a),
+  :global(a *)
+    @apply text-accent1-default
```
Comment (georgeciubotaru) [inline src/lib/components/BodyRenderer/Blocks/link.svelte:62]:
```suggestion
  a :global(*)
    @apply text-inherit
```

File: src/lib/components/BodyParser/blocks.ts [hunk 3]
```diff
@@ -247,7 +247,7 @@ const tokeniseInlineEls = (inlineBlocks: string[]) => {
       const match = b.match(boldExp) as RegExpExecArray;
       tokens.push({
         type: 'inline',
-        text: `<b>${match[1]}</b>`,
+        text: `<b>${match[1]}</b>`, // redact here (this file)? (test)
```
Comment (georgeciubotaru) [inline src/lib/components/BodyParser/blocks.ts:250]:
```suggestion
        text: `<b>${match[1]}</b>`,
```

File: src/lib/components/BodyParser/blocks.ts [hunk 4]
```diff
@@ -271,7 +271,7 @@ const tokeniseInlineEls = (inlineBlocks: string[]) => {
       const match = b.match(underlineExp) as RegExpExecArray;
       tokens.push({
         type: 'inline',
-        text: `<b>${match[1]}</b>`,
+        text: `<b>${match[1]}</b>`, // redact here (this file)? (test)
```
Comment (georgeciubotaru) [inline src/lib/components/BodyParser/blocks.ts:274]:
```suggestion
        text: `<b>${match[1]}</b>`,
```

**Review by georgeciubotaru — CHANGES_REQUESTED** [review]
see my change

**Review by georgeciubotaru — APPROVED** [review]

**Pull request merged**